pkg/selectors/kernel.go

```diff
@@ -1181,6 +1182,16 @@ func ParseMatchBinary(k *KernelSelectorState, b *v1alpha1.BinarySelector, selIdx
 	// prepare the selector options
 	sel := MatchBinariesSelectorOptions{}
 	sel.Op = op
+	sel.MBSetID = mbset.InvalidID
+	if b.FollowChildren {
+		if op != SelectorOpIn {
+			return fmt.Errorf("matchBinary: followChildren not yet implemented for operation '%s'", b.Operator)
+		}
+		sel.MBSetID, err = mbset.AllocID()
+		if err != nil {
+			return fmt.Errorf("matchBinary: %w", err)
```

Comment:
nit: you can add something like `matchBinary: followChildren failed to allocate ID: %w`

Reply:
fixed, thanks!